tools/check-wininstaller-translations.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -eu
+
+BASE_DIR=src/res/translation/wininstaller/
+BASE_LANG=en
+INSTALLERLNG=installerlng.nsi
+BASE_LANGSTRINGS=$(grep LangString "${BASE_DIR}/${BASE_LANG}.nsi" | cut -d' ' -f2)
+EXIT=0
+LANGUAGE_FILES="$(ls -1 src/res/translation/wininstaller/??.nsi | grep -vF "${BASE_LANG}.nsi")"
+for LANGUAGE_FILE in ${LANGUAGE_FILES}; do
+    echo
+    echo "* ${LANGUAGE_FILE}"
+    echo -n "  - Checking language file is included in ${INSTALLERLNG}... "
+    if grep -qF $(tr '/' '\\' <<<"${LANGUAGE_FILE}") "${BASE_DIR}/${INSTALLERLNG}"; then
+        echo "ok"
+    else
+        echo "ERROR"
+        EXIT=1
+    fi
+
+    echo -n "  - Checking language file for empty strings... "
+    if grep -qF '""' "${LANGUAGE_FILE}"; then
+        echo "ERROR, found empty string(s)"
```

Comment:
I'm not sure if we want to output an error here or just a warning?

Reply:
Ok, changed to warning.